content/en/apps/reference/api.md

```diff
@@ -1132,7 +1134,76 @@ curl http://localhost:5988/api/v1/monitoring
+
+Available as of 3.12.x.
+Used to retrieve a range of metrics about the instance. While the output is human-readable this is intended for automated monitoring allowing for tracking trends over time and alerting about potential issues.
+
+#### Permissions
+
+No permissions required.
+
+#### Examples
+
+##### JSON format
+
+```
+curl http://localhost:5988/api/v2/monitoring
+
+{"version":{"app":"3.12.0","node":"v10.16.0","couchdb":"2.3.1"},"couchdb":{"medic":{"name":"medic","update_sequence":5733,"doc_count":278,"doc_del_count":32,"fragmentation":1.0283517758420173}...
+```
+
+#### Response content
+
+| JSON path | Type | Description |
+| --------- | ----------------- | ---- |
+| `version.app` | String | The version of the webapp. |
+| `version.node` | String | The version of NodeJS. |
+| `version.couchdb` | String | The version of CouchDB. |
+| `couchdb.<dbname>.name` | String | The name of the db, usually one of "medic", "medic-sentinel", "medic-users-meta", "_users". |
+| `couchdb.<dbname>.update_sequence` | Number | The number of changes in the db. |
+| `couchdb.<dbname>.doc_count` | Number | The number of docs in the db. |
+| `couchdb.<dbname>.doc_del_count` | Number | The number of deleted docs in the db. |
+| `couchdb.<dbname>.fragmentation` |  Number | The fragmentation of the db, lower is better, "1" is no fragmentation. |
+| `date.current` | Number | The current server date in millis since the epoch, useful for ensuring the server time is correct. |
+| `date.uptime` | Number | How long API has been running. |
+| `sentinel.backlog` | Number | Number of changes yet to be processed by Sentinel. |
+| `messaging.outgoing.total.due` | Number | The number of messages due to be sent. |
+| `messaging.outgoing.total.scheduled` | Number | The number of messages scheduled to be sent in the future. |
+| `messaging.outgoing.total.muted` | Number | The number of messages that are muted and therefore will not be sent. |
+| `messaging.outgoing.total.delivered` | Number | The number of messages that have been delivered or sent. |
+| `messaging.outgoing.total.failed` | Number | The number of messages have failed to be delivered. |
+| `messaging.outgoing.seven_days.due` | Number | The number of messages due to be sent in the last seven days. |
+| `messaging.outgoing.seven_days.scheduled` | Number | The number of messages that were scheduled to be sent in the last seven days. |
+| `messaging.outgoing.seven_days.muted` | Number | The number of messages that were due in the last seven days and are muted. |
+| `messaging.outgoing.seven_days.delivered` | Number | The number of messages that were due in the last seven days and have been delivered or sent. |
+| `messaging.outgoing.seven_days.failed` | Number | The number of messages that were due in the last seven days and have failed to be delivered. |
+| `messaging.outgoing.last_hundred.pending` | Object | Counts within last 100 messages that have received status updates, and are one of the "pending" statuses | 
```

Comment:
```suggestion
| `messaging.outgoing.last_hundred.pending` | Object | Counts within last 100 messages that have received status updates, and are one of the "pending" group statuses | 
```
Not sure to make clear between: pending the group of statuses and pending the status, same with muted group